gap/PolygonalComplexes/graphs.gd

```diff
@@ -1099,3 +1099,81 @@ DeclareOperation( "OnVertexEdgePaths",
 DeclareOperation( "OnEdgeFacePaths", 
     [ IsEdgeFacePath , IsPerm ] );
 #! @EndGroup OnEdgeFacePaths
+
+#! @BeginGroup ButterflyInsertion

```

Comment:
The name ButterflyInsertion already exists. So maybe everything should be renamed to `EdgeAddition` and `EdgeDeletion` (these are the common names for these operations on graphs).